packages/benchmarks/tsconfig.json

```diff
@@ -5,5 +5,5 @@
     "module": "NodeNext",
     "moduleResolution": "NodeNext"
   },
-  "include": ["./src/**/*.ts"]
+  "include": ["./src/**/*.ts", "../mongodb-rag-core/src/models/models.test.ts", "../mongodb-rag-core/src/models/models.ts", "../mongodb-rag-core/src/models/makeOpenAiClientFactory.ts"]
```

Comment:
Why include these references in `benchmarks` vs update the `mongodb-rag-core` tsconfig and just import the built lib?

Reply:
thats a mistake. sometimes autohapepns when move files..idk why. will remove